src/ParallelAlgorithms/Events/ObserveAtExtremum.hpp

```diff
@@ -160,11 +144,11 @@ class ObserveAtExtremum<ObservationValueTag,
 
   ObserveAtExtremum() = default;
 
-  ObserveAtExtremum(const std::string& subfile_name,
-                    const ObserveTensors& observe_tensors);
+  ObserveAtExtremum(std::string subfile_name,
+                    ObserveTensors observe_tensors);
 
-  using observed_reduction_data_tags =
-      observers::make_reduction_data_tags<tmpl::list<ReductionData>>;
+  // using observed_reduction_data_tags =
+  //     observers::make_reduction_data_tags<tmpl::list<ReductionData>>;
```

Comment:
I'm not entirely sure what this alias is used for, but I think it's for registering the possible reductions with charm or our wrappers or something.  The argument should probably be `tmpl::list<ReductionData<funcl::Max<>>, ReductionData<funcl::Min<>>>`.

Reply:
Ok, that makes sense from what I had read; I guess that part will only be tested once we actually plug this observer in a simulation doing a real reduction.